canvas2D/src/Engine/babylon.prim2dBase.ts

```diff
@@ -1620,10 +1620,7 @@
             }
 
             if (settings.layoutData) {
-                let p = this.parent;
-                if (p && p.layoutEngine) {
-                    p.layoutEngine.newChild(this, settings.layoutData);
-                }
+                this.layoutData = settings.layoutData;

```

Comment:
Yes, much cleaver that what I did, so you removed the newChild() method in LayoutEngineBase?  If not, could you?